nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/PutKinesisStream.java

```diff
@@ -76,17 +77,15 @@ public class PutKinesisStream extends AbstractAwsSyncProcessor<KinesisClient, Ki
     public static final String AWS_KINESIS_SEQUENCE_NUMBER = "aws.kinesis.sequence.number";
 
     public static final PropertyDescriptor KINESIS_PARTITION_KEY = new PropertyDescriptor.Builder()
-        .displayName("Amazon Kinesis Stream Partition Key")
-        .name("amazon-kinesis-stream-partition-key")
+        .name("Amazon Kinesis Stream Partition Key")
```

Comment:
"Amazon Kinesis" is redundant since it is in the context of the Processor.
```suggestion
        .name("Stream Partition Key")
```